src/card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 0;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-family: "Roboto", sans-serif;
```

Comment:
Redundant, just add font-family to `.card` and all others inherit it. Remove font-family in all child elements